README.md

```diff
@@ -243,11 +153,63 @@ async def transform(msg):
 ```
 
 Dispatch converts Python coroutines to *Distributed Coroutines*, which can be
-suspended and resumed on any instance of a service across a fleet.
+suspended and resumed on any instance of a service across a fleet. For a deep
+dive on these concepts, read our blog post on
+[*Distributed Coroutines with a Native Python Extension and Dispatch*](https://stealthrocket.tech/blog/distributed-coroutines-in-python).
+
+### Integration with FastAPI
+
+Many web applications written in Python are developed using [FastAPI][fastapi].
+Dispatch can integrate with these applications by instantiating a
+`dispatch.fastapi.Dispatch` object. When doing so, the Dispatch functions
+declared by the program can be invoked remotely over the same HTTP interface
+used for the [FastAPI][fastapi] handlers.
+
+The following code snippet is a complete example showing how to install a
+`Dispatch` instance on a [FastAPI][fastapi] server:
+
+```python
+from fastapi import FastAPI
+from dispatch.fastapi import Dispatch
+import requests
+
+app = FastAPI()
+dispatch = Dispatch(app)
+
+@dispatch.function
+def publish(url, payload):
+    r = requests.post(url, data=payload)
+    r.raise_for_status()
+
+@app.get('/')
+def root():
+    publish.dispatch('https://httpstat.us/200', {'hello': 'world'})
+    return {'answer': 42}
+```
+
+In this example, GET requests on the HTTP server dispatch calls to the
+`publish` function. The function runs concurrently to the rest of the
+program, driven by the Dispatch SDK.
+
+### Configuration
+
+In order for Dispatch to interact with functions remotely, the SDK needs to be
+configured with the address at which the server can be reached. The Dispatch
+API Key must also be set, and optionally, a public signing key should be
+configured to verify that requests originated from Dispatch. These
+configuration options can be passed as arguments to the
+the `Dispatch` constructor, but by default they will be loaded from environment
+variables:
+
+| Environment Variable        | Value Example                      |
+| :-------------------------- | :--------------------------------- |
+| `DISPATCH_API_KEY`          | `d4caSl21a5wdx5AxMjdaMeWehaIyXVnN` |
+| `DISPATCH_ENDPOINT_URL`     | `https://service.domain.com`       |
+| `DISPATCH_VERIFICATION_KEY` | `-----BEGIN PUBLIC KEY-----...`    |
```

Comment:
Maybe a note here that you don't need to populate these when using the CLI since it does it automatically?